css-values-4/Overview.bs

```diff
@@ -2918,6 +2918,75 @@ Combination of <<position>></h4>
 	as a <<length-percentage>>.
 
 
+<!--
+████████ ████████     ███      ██████  ████████ ████  ███████  ██    ██
+██       ██     ██   ██ ██    ██    ██    ██     ██  ██     ██ ███   ██
+██       ██     ██  ██   ██   ██          ██     ██  ██     ██ ████  ██
+██████   ████████  ██     ██  ██          ██     ██  ██     ██ ██ ██ ██
+██       ██   ██   █████████  ██          ██     ██  ██     ██ ██  ████
+██       ██    ██  ██     ██  ██    ██    ██     ██  ██     ██ ██   ███
+██       ██     ██ ██     ██   ██████     ██    ████  ███████  ██    ██
+-->
+
+<h3 id="fractions">
+Flexible lengths: the <<fraction>> type</h3>
+
+	Fraction values are non-negative <<dimension>>s denoted by
```

Comment:
I would start with *Fraction values are denoted by `<fraction>`, and indicates a value that is some fraction of another reference value...*, borrowing the definition of [`<percentage>`](https://drafts.csswg.org/css-values-4/#percentages), then by emphasizing the difference between `<fraction>` and `<percentage>`.

Then I would say: *A fraction is a [dimension](https://drafts.csswg.org/css-values-4/#dimension)*, borrowing the definition of `<length>`.

Reply:
And finally with *the allowed range of `<fraction>` values always excludes negative values, in addition to any explicit ranges that might be specified*, borrowing the definition of `<resolution>`.

Reply:
Technically the math *could* support negative fractions, though not sure there are enough use cases to be worth the complexity.

Reply:
I'm not sure what the meaning of a negative fraction would be; even if there is a consistent "just follow the numbers" behavior one could assign to it, I wouldn't want to make it valid without a reasonable argument that it's actually doing something useful and understandable.

Reply:
I've rephrased the description.

@LeaVerou Please open a separate issue for discussion negative fractions if you see a use case for them.

The change here is only meant to centralize and generalize the definition for the existing usages.